src/index.mjs

```diff
@@ -0,0 +1,51 @@
+export { default as chain } from './chain'
+export { default as combinationsWithReplacement } from './combinations-with-replacement'
+export { default as combinations } from './combinations'
+export { default as compose } from './compose'
+export { default as compress } from './compress'
+export { default as count } from './count'
+export { default as cycle } from './cycle'
+export { default as dropWhile } from './drop-while'
+export { default as enumerate } from './enumerate'
+export { default as execute } from './execute'
+export { default as filter } from './filter'
+export { default as flatMap } from './flat-map'
+export { default as groupBy } from './group-by'
+export { default as iter } from './iter'
+export { default as map } from './map'
+export { default as permutations } from './permutations'
+export { default as product } from './product'
+export { default as range } from './range'
+export { default as reduceIter } from './reduce-iter'
+export { default as reduce } from './reduce'
+export { default as regexpExec } from './regexp-exec'
+export { default as regexpSplit } from './regexp-split'
+export { default as regexpSplitIter } from './regexp-split-iter'
+export { default as splitLines } from './split-lines'
+export { default as repeat } from './repeat'
+export { default as slice } from './slice'
+export { default as takeWhile } from './take-while'
+export { default as tee } from './tee'
+export { default as zipLongest } from './zip-longest'
+export { default as zip } from './zip'
+export { default as asyncIterToArray } from './async/async-iter-to-array'
```

Comment:
It might make sense to put these in ```async/index```. Quite a few people are likely to use the library only for the sync versions, so I don't think it's that big a burden to have to write:
```import asyncMap from 'iter-tools/async';```